ui/plugins/controller.js

```diff
@@ -183,6 +183,12 @@ treeherder.controller('PluginCtrl', [
                             }
                             return result;
                         }, []);
+                        $scope.buildernameIndex = 0;
+                        $scope.job_details.forEach(function(job, i) {
+                            if (job.title === "Buildername") {
+                                $scope.buildernameIndex = i;
+                            }
```

Comment:
You should be able to replace this with:

```js
$scope.buildernameIndex = _.findIndex($scope.job_details, {title: "BuilderName"}
```

See: https://github.com/lodash/lodash/blob/3.10.1/doc/README.md#_findindexarray-predicate_identity-thisarg